---
layout: default
---
# BioC 2018: Where Software and Biology Connect

When: July 25 (Developer Day) 26 and 27, 2018 <br />
Where: [Victoria University][uvic], University of Toronto, Toronto, Canada

[uvic]: http://www.vicu.utoronto.ca/

## Schedule (Tentative)

Confirmed Speakers for July 26-27

* [Michael Hoffman][5], University of Toronto. Identifying
  transcription factor binding using open chromatin, transcriptome,
  and methylation data.
* [Elana Fertig][3], John Hopkins Medicine Division of Biostatistics
  and Bioinformatics. Enter the Matrix: Interpreting omics through
  matrix factorization.
* [Brenda Andrews][1], University of Toronto.  Analysis of high
  content microscopy data generated through automated yeast genetics.
* [Tim Hughes][6], University of Toronto. Binding motifs for DNA and
  RNA binding proteins.
* [Benjamin Haibe-Kains][2], Princesss Margaret Cancer Center,
  Toronto. Cancer Biomarker Discovery: Building a Bridge Between
  Preclinical and Clinical Research.
* [Charlotte Soneson][4], University of Zurich, Switzerland. A
  junction coverage compatibility score to quantify the reliability of
  transcript abundance estimates and annotation catalogs.

[1]: http://sites.utoronto.ca/andrewslab/
[2]: https://www.pmgenomics.ca/bhklab/
[3]: https://www.rits.onc.jhmi.edu/DBB/members/?members=Faculty&member=efertig1
[4]: https://csoneson.github.io/
[5]: https://hoffmanlab.org/
[6]: http://hugheslab.med.utoronto.ca/

Developer Day: Wednesday July 25

* See the [Developer Day](schedule-developer-day) schedule.
   
Main Conference Day 1: Thursday July 26

* 8:00 - 8:30 REGISTRATION
* 8:30 - 9:00 Welcoming remarks (Martin Morgan)
* 9:00 - 9:30 Michael Hoffman
* 9:30 - 10:00 Elana Fertig
* 10:00 - 10:30 break
* 10:30 - 11:00 Brenda Andrews
* 11:00 - 12:00 Contributed talks
    + Innes BT\* and Bader GD. scClustViz - Single-cell RNAseq Cluster
      Assessment and Interactive Visualisation.
    + Nima Hejazi\*, Alan Hubbard, Mark van der Laan. Data-Adaptive
      Estimation and Inference for Differential Methylation Analysis.
    + Adithya M, Bhargava A, Wright E\*. Improving the accuracy of
      taxonomic classification for identifying taxa in microbiome
      samples.
    + Zhun Miao\*, Ke Deng, Xiaowo Wang, Xuegong Zhang. DEsingle for
      detecting three types of differential expression in single-cell
      RNA-seq data.
    + Ludwig Geistlinger\*, Gergely Csaba, Mara Santarelli, Lucas
      Schiffer, Marcel Ramos, Ralf Zimmer, and Levi Waldron. Towards a
      gold standard for benchmarking gene set enrichment analysis.
* 12:00 - 1:00 Lunch / Birds-of-a-feather
* 1:00 - 2:45 Workshop Session 1
    + Martin Morgan, Bioconductor for Everyone
    + Ludwig Geistlinger* and Levi Waldron,	Functional enrichment analysis of high-throughput omics data
* 1:00 - 1:45 Workshops Session 1a
* 2:00 - 2:45 Workshops Session 1b
    + Zhaleh Safikhani*, Petr Smirnov, Benjamin Haibe-Kains	Biomarker discovery from large pharmacogenomics datasets
* 2:45 - 3:00 break
* 3:15 - 5:00 Workshops Session 2
    + MacDonald J, Shepherd L	Introduction to Bioconductor annotation resources
    + Love MI	RNA-seq data analysis with DESeq2
* 3:15 - 4:00 Workshops Session 2a
    + Coetzee SG* and Hazelett DJ	Variant Functional Annotation using StatePaintR, FunciVar and MotifBreakR
* 4:15 - 5:00 Workshops Session 2b
     + Isserlin R*,Innes B, Bader GD	Cytoscape Automation in R using Rcy3
* 5:30 - 7:30 Contributed posters

Main Conference Day 2: Friday July 27

* 8:30 - 9:00 Tim Hughes
* 9:00 - 9:30 Benjamin Haibe-Kains
* 9:30 - 10:00 Contributed talks
    + Lee S\*, Cook D, Lawrence M. plyranges: a fluent interface to
      Bioconductor's Ranges infrastructure.
    + Love MI\*, Hickey P, Soneson, C, and Patro R. Automatic metadata
      propagation for RNA-seq.
* 10:00 - 10:30 break
* 10:30 - 11:00 Charlotte Soneson
* 11:00 - 12:00 Contributed talks
    + Albert Y Zhang, Shian Su, Matthew E Ritchie, and Charity W
      Law\*. Unpacking signal from RNA-seq intron reads using Rsubread
      and limma packages.
    + Steinbaugh MJ\*, Kirchner RD, Ho Sui S. bcbioSingleCell: R
      package for bcbio single-cell analysis.
    + Abbas Rizvi\*, Ezgi Karaesmen\*, Leah Preus, Michael Sovic,
      Junke Wang, Lara Sucheston-Campbell. gwasurvivr: an R package to
      perform survival association testing on imputed genetic data.
    + Righelli D\*, Koberstein J, Gomes B, Zhang N, Angelini C,
      Peixoto L, Risso D. Differential Enriched Scan 2 (DEScan2): a
      fast pipeline for broad peak analysis.
    + Rachael V Phillips\*, Alan Hubbard. Data Adaptive Evaluation of
      Preprocessing Methods using Ensemble Machine Learning.
* 12:00 - 1:00 Lunch / Birds-of-a-feather
* 1:00 - 2:45 Workshops Session 1
     + Michael Lawrence*, Martin Morgan	Solving common bioinformatic challenges using GenomicRanges
     + Ramos M, Geistlinger L, Waldron L*	Workflow for Multi-omics Analysis with MultiAssayExperiment
* 2:00 - 2:45 Workshops Session 1b
     + Charity Law* , Monther Alhamdoosh, Shian Su, Gordon Smyth and Matthew Ritchie 	RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR
* 2:45 - 3:00 break
* 3:15 - 5:00 Workshops Session 2
     + Das D*, Street K*, Risso D*	Analysis of single-cell RNA-seq data: Dimensionality reduction, clustering, and lineage inference
     + Wright E	Working with Genomic Data in R with the DECIPHER package
     + Levi Waldron, Benjamin Haibe-Kains, Sean Davis	Public Data Resources and Bioconductor
* 4:15 - 5:00 Workshops Session 2b
     + Lee S*, Lawrence M	Fluent genomic data analysis with plyranges
* 5:30 - 7:30 Contributed posters
